### PR TITLE
Fixes thermite burning lava

### LINKED ISSUE #32977: thermite can melt lava
https://github.com/HippieStation/HippieStation/issues/4978

@ninjanomnom 

so jet fuel can't melt steel beams but thermite can melt lava.

### PR DESCRIPTION
**Pull request by ninjanomnom**
:cl: ninjanomnom
fix: Fixes thermite burning hotter than the boiling point of stone
/:cl:

fixes #32977

### CODE REVIEW

**Comment by Cyberboss** [comment]
Duplicate of #32980 >:)

**Comment by Buggy123** [comment]
Feature.

File: code/datums/components/thermite.dm [hunk 1]
```diff
@@ -3,8 +3,17 @@
 	var/amount
 	var/overlay
 
-	var/static/list/blacklist = typecacheof(/turf/closed/wall/mineral/diamond)
-	var/static/list/resistlist = typecacheof(/turf/closed/wall/r_wall)
+	var/static/list/blacklist = typecacheof(
+		/turf/closed/wall/mineral/diamond,
+		/turf/closed/indestructible,
+		/turf/open/lava,
```
Comment (ShizCalev) [inline code/datums/components/thermite.dm]: image

/turf/open/indestructible
/turf/open/space

**Review by ShizCalev — DISMISSED** [review]

File: code/datums/components/thermite.dm (hunk 1 above)
Comment (ninjanomnom) [reply code/datums/components/thermite.dm]: space just becomes space again, it still burns even with blacklist. It just doesn't get change turf'd

Comment (ninjanomnom) [reply code/datums/components/thermite.dm]: I'll make an actual blocked list

**Pull request merged**